apps/meteor/app/importer-csv/server/addParsedContacts.ts

```diff
@@ -0,0 +1,36 @@
+import { Random } from '@rocket.chat/random';
+
+import type { ImportDataConverter } from '../../importer/server/classes/ImportDataConverter';
+
+export async function addParsedContacts(this: ImportDataConverter, parsedContacts: string[][]): Promise<number> {
```

Comment:
It would be better to move this as the importer class method instead of providing the `this` for the function.

Reply:
I did it here just to keep the two PRs independent from each other; I'll still be opening a third one with the actual code to convert the contacts and I'll be moving this function on it.